enterprise/server/githubapp/githubapp.go

```diff
@@ -414,10 +416,45 @@ func (a *GitHubApp) maybeTriggerBuildBuddyWorkflow(ctx context.Context, eventTyp
 	if err != nil {
 		return err
 	}
+	if err := a.MaybeReindexRepo(ctx, row.GitRepository, wd, repoURL, tok.GetToken()); err != nil {
+		log.Debugf("Not indexing repo: %s", err)
+	}
```

Comment:
This is the main significant change: if we receive a webhook, we check if the user has codesearch enabled and possibly send an async RPC to the codesearch endpoint to re-index a repo.